modules/apps/style-book/style-book-web/src/main/java/com/liferay/style/book/web/internal/display/context/StyleBookManagementToolbarDisplayContext.java

```diff
@@ -126,6 +144,46 @@ public Map<String, Object> getAdditionalProps() {
 
 				return exportStyleBookEntriesURL.toString();
 			}
+		).put(
+			"frontendTokenDefinitions",
+			() -> {
+				long companyId = _themeDisplay.getCompanyId();
+
+				List<Object> frontendTokenDefinitionProviders =
```

Comment:
@SelenaAungst When we declare a list on the backend, we always put its type. In this case, the elements being added are of type `Map<String, Object>`. We only use `Object` when either the list has more than one type or you are not sure which type is this going to be.

Reply:
Just to confirm, this should be: 
List<Map<String, Object>> frontendTokenDefinitionProviders = new ArrayList<>();